sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -0,0 +1,1988 @@
+    getWorkspace: (requestParams?: OptionalRequestParameters) => Promise<GetWorkspaceResponseOutput>;
+    listConnections: (options?: ListConnectionsQueryParamProperties, requestParams?: OptionalRequestParameters) => Promise<Array<GetConnectionResponseOutput>>;
+}
+
+// @public
+export enum connectionToolType {
+    BingGrounding = "bing_grounding",
+    MicrosoftFabric = "microsoft_fabric",
+    SharepointGrounding = "sharepoint_grounding"
+}
+
+// @public
+export type ConnectionType = "AzureOpenAI" | "Serverless" | "AzureBlob" | "AIServices" | "CognitiveSearch";
+
+// @public
+export type ConnectionTypeOutput = "AzureOpenAI" | "Serverless" | "AzureBlob" | "AIServices" | "CognitiveSearch";
+
+// @public
+export interface CreateAgentOptions {
+    description?: string | null;
+    instructions?: string | null;
+    metadata?: Record<string, string> | null;
+    model: string;
+    name?: string | null;
+    response_format?: AgentsApiResponseFormatOption | null;
+    temperature?: number | null;
+    tool_resources?: ToolResources | null;
+    tools?: Array<ToolDefinition>;
+    top_p?: number | null;
+}
+
+// @public
+export interface CreateAndRunThreadOptions {
+    assistant_id: string;
+    instructions?: string | null;
+    max_completion_tokens?: number | null;
+    max_prompt_tokens?: number | null;
+    metadata?: Record<string, string> | null;
+    model?: string | null;
+    response_format?: AgentsApiResponseFormatOption | null;
+    stream?: boolean;
+    temperature?: number | null;
+    thread?: AgentThreadCreationOptions;
+    tool_choice?: AgentsApiToolChoiceOption | null;
+    tool_resources?: UpdateToolResourcesOptions | null;
+    tools?: Array<ToolDefinition> | null;
+    top_p?: number | null;
+    truncation_strategy?: TruncationObject | null;
+}
+
+// @public
+export interface CreateRunOptions {
+    additional_instructions?: string | null;
+    additional_messages?: Array<ThreadMessage> | null;
+    assistant_id: string;
+    instructions?: string | null;
+    max_completion_tokens?: number | null;
+    max_prompt_tokens?: number | null;
+    metadata?: Record<string, string> | null;
+    model?: string | null;
+    response_format?: AgentsApiResponseFormatOption | null;
+    stream?: boolean;
+    temperature?: number | null;
+    tool_choice?: AgentsApiToolChoiceOption | null;
+    tools?: Array<ToolDefinition> | null;
+    top_p?: number | null;
+    truncation_strategy?: TruncationObject | null;
+}
+
+// @public
+export interface CreateVectorStoreFileBatchOptions {
+    chunkingStrategy?: VectorStoreChunkingStrategyRequest;
+    dataSources?: VectorStoreDataSource[];
+    fileIds?: string[];
+}
+
+// @public
+export interface CreateVectorStoreFileOptions {
+    chunkingStrategy?: VectorStoreChunkingStrategyRequest;
+    dataSources?: Array<VectorStoreDataSource>;
+    fileId?: string;
+}
+
+// @public
+export interface CredentialsApiKeyAuthOutput {
```

Comment:
We have KeyCredential and TokenCredential APIs defined in core-auth - would these not work here?